bolt-design-system/bolt
# Add action base class

## Jira

 - http://vjira2:8080/browse/BDS-534

## Summary

Add a base class that contains the shared parts of links and buttons, and extend *that* when adding new "actionable" components, rather than extending the link or button components themselves.

## Details

TBD

## How to test

See [files changed](https://github.com/bolt-design-system/bolt/pull/1000/files). I'm looking for feedback on:
 - method of setting `rootElementTags`
 - should this be in renderers or make it and element called `action`?
 - should I try to refactor the chunks of the `render` method into something reusable (handling slots perhaps)? Not worth it?
